src/app/pages/swap/swap-summary/swap-summary-tabs.tsx

```diff
@@ -0,0 +1,62 @@
+import { Suspense, useState } from 'react';
+
+import { Box, Flex, SlideFade, Stack } from '@stacks/ui';
+import type { StackProps } from '@stacks/ui';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { LoadingSpinner } from '@app/components/loading-spinner';
+import { Tabs } from '@app/components/tabs';
+
+const analyticsPath = ['/recommended', '/custom'];
+
+interface SwapSummaryTabsProps extends StackProps {
+  details: React.JSX.Element;
+  status: React.JSX.Element;
+}
+export function SwapSummaryTabs(props: SwapSummaryTabsProps) {
+  const { details, status, ...rest } = props;
+  const analytics = useAnalytics();
+  const [activeTab, setActiveTab] = useState(0);
```

Comment:
It would be good to use routes for tabs, like I added here: https://github.com/hirosystems/wallet/pull/4001 

I can refactor this though when finishing the routing changes I have here: https://github.com/hirosystems/wallet/pull/4099